### DISCUSSION TITLE
Support `time-rs` as an alternative to `chrono`

### DISCUSSION THREAD

**Discussion by LouisGariepy** [SeaQL/sea-orm#359] [Q&A] [answered]
Sqlx makes it possible to use `time-rs` which seems to be superseding `chrono` anyway now. Would it be possible to add support for it in SeaOrm? Thanks!

Edit: By the way, I think what you're doing here is amazing and has much potentential and value for the Rust community. Thank you.

**Comment by billy1624** [comment, marked as answer]
Thanks for the love and support! We will look into it.

**Reply by LouisGariepy** [reply]
Thank you!
